lib/main_common.pm

```diff
@@ -1283,6 +1283,7 @@ sub load_yast2_ncurses_tests {
     if (get_var('Y2UITEST_DEVEL')) {
         # (Livesystem and laptops do use networkmanager)
         if (!get_var("LIVETEST") && !get_var("LAPTOP")) {
+            loadtest 'console/zypper_info';
```

Comment:
That does not seem to fit here at all. I doubt we want to call a "zypper info" test as part of "YaST UI test development" group. That does not sound like it would go well together, sorry

Reply:
It does, because zypper info adds repo-source. Otherwise we encounter below error.
http://dhcp151.suse.cz/tests/3361#step/yast2_cmdline/6

Reply:
I guess it makes sense if we either extract adding source repos to some util function and call in both places. Not sure if we should put either zypper_info here or vice versa this test module after zypper_info.